### PR TITLE
Ensure CNI dir exists before writing openshift CNI configuration under CNI dir

### PR DESCRIPTION
**Pull request by pravisankar**
Found this issue on RHEL containerized openshift install.

Fixes: https://github.com/openshift/origin/issues/14661
Fixes: https://github.com/openshift/origin/issues/15147

### CODE REVIEW

**Comment by pravisankar** [comment]
@openshift/networking @dcbw PTAL

File: pkg/sdn/plugin/node.go [hunk 1]
```diff
@@ -326,13 +332,17 @@ func (node *OsdnNode) Start() error {
 		}
 	}
 
+	if err := os.MkdirAll(CNI_DIR_PATH, os.ModePerm); err != nil {
```
Comment (rajatchopra) [inline pkg/sdn/plugin/node.go]: FileMode could be 0755 maybe?

**Comment by pravisankar** [comment]
[test]

File: pkg/sdn/plugin/node.go [hunk 2]
```diff
@@ -38,6 +39,11 @@ import (
 	kexec "k8s.io/kubernetes/pkg/util/exec"
 )
 
+const (
+	CNI_DIR_PATH           = "/etc/cni/net.d"
```
Comment (smarterclayton) [inline pkg/sdn/plugin/node.go]: We don't do constants like this - if this is only used in this package, should be `camelCase`

**Comment by pravisankar** [comment]
Changed constants to `camelCase`

**Comment by smarterclayton** [comment]
[test]

**Pull request merged**